AsyncAwait.Task2.CodeReviewChallenge/Middleware/StatisticMiddleware.cs

```diff
@@ -23,20 +23,23 @@ public async Task InvokeAsync(HttpContext context)
     {
         string path = context.Request.Path;
 
-        var staticRegTask = Task.Run(
-            () => _statisticService.RegisterVisitAsync(path)
-                .ConfigureAwait(false)
-                .GetAwaiter().OnCompleted(UpdateHeaders));
+        Task staticRegTask = Task.Run(
```

Comment:
Why do you need a separate task here?

Reply:
I'm waiting for task finishing below in code.

Reply:
What is the benefit of starting a new task anyway?